internal/rpc/health/errors.go

```diff
@@ -0,0 +1,25 @@
+package health
+
+type Error struct {
+	Code    int
+	Message string
+}
+
+func (s Error) Error() string {
+	return s.Message
+}
+
+var (
+	NodeNotSyncing = Error{
+		Code:    10,
+		Message: "Node is not syncing",
+	}
+	VMNotRunning = Error{
+		Code:    20,
+		Message: "Cairo VM is not running",
+	}
+	UnHealthy = Error{
+		Code:    30,
+		Message: "Node is not healthy",
+	}
+)
```

Comment:
Const should be used with iota. Each error should be prefixed with `Err...` which is idiomatic golang

Reply:
I did the fix with the Err, but not the iota, we are handling the errors in that way on the starknet rpc handler and I prefer to keep it in that way for this PR, and maybe later change both places